metadata-models/src/main/pegasus/com/linkedin/metadata/aspect/MLModelEndpointAspect.pdl

```diff
@@ -0,0 +1,20 @@
+namespace com.linkedin.metadata.aspect
+
+import com.linkedin.metadata.key.MLModelEndpointKey
+import com.linkedin.ml.metadata.MLModelEndpointProperties
+import com.linkedin.common.Ownership
+import com.linkedin.common.Status
+import com.linkedin.common.Deprecation
+import com.linkedin.common.BrowsePaths
+
+/**
+ * A union of all supported metadata aspects for a MLModelEndpoint
+ */
+typeref MLModelEndpointAspect = union[
+  MLModelEndpointKey,
+  MLModelEndpointProperties,
+  Ownership,
+  Status,
+  Deprecation,
+  BrowsePaths
```

Comment:
will ml model endpoints be browsable? probably not, right? I imagine these will just be linked via ml models